lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -533,9 +628,22 @@ class TraceProcessor {
     // Find the inspected frame
     const mainFrameIds = this.findMainFrameIds(keyEvents);
 
+    const frames = keyEvents
```

Comment:
maybe `resolveRootFrames` could take `keyEvents` and do this part itself? would that make the tests more complicated?

Reply:
Frames is used in the return value of `computeTraceOfTab`. I just moved it up so it could be used to find LCP all frames.